tests/test_util/test_warning.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+"""Unit tests for union_dict.
+"""
+from unittest import TestCase, main
+
+from cogent3.util.warning import deprecate
+
+
+__author__ = "Richard Morris"
+__copyright__ = "Copyright 2007-2022, The Cogent Project"
+__credits__ = ["Gavin Huttley", "Richard Morris"]
+__license__ = "BSD-3"
+__version__ = "2023.2.12a1"
+__maintainer__ = "Gavin Huttley"
+__email__ = "gavin.huttley@anu.edu.au"
+__status__ = "Production"
+
+
+class WarningTests(TestCase):
```

Comment:
don't use `unittest.TestCase`, just make valinall `test_<something> functions which will be exercised by `pytest`.